saleor/product/models.py

```diff
@@ -87,14 +87,26 @@ def __str__(self) -> str:
         return self.name
 
 
-class CategoryTranslation(SeoModelTranslation):
+class CategoryTranslation(SeoModelTranslationWithSlug):
     category = models.ForeignKey(
         Category, related_name="translations", on_delete=models.CASCADE
     )
     name = models.CharField(max_length=128, blank=True, null=True)
     description = SanitizedJSONField(blank=True, null=True, sanitizer=clean_editor_js)
 
     class Meta:
+        indexes = [
```

Comment:
Just one small thing from me, this definition is redundant cause we know of this index by defining unique constraint.
So we can remove it from here, and from migration state operations (but we leave of course the database operation of index creation)

Reply:
Thanks for the feedback - fixed